src/fluree/db/peer/http_api.clj

```diff
@@ -779,10 +779,10 @@
                                            (throw (ex-info (str "If using a closed api, a storage request must be returned as json.")
                                                            {:status 401
                                                             :error  :db/invalid-transaction})))
-                        {:keys [network ledger type key]} params
-                        ledger           (keyword network ledger)
-                        _                (when-not (and network ledger type)
-                                           (throw (ex-info (str "Incomplete request. At least a network, db and type are required. Provided network: " network " ledger: " ledger " type: " type " key: " key)
+                        {network :network dbid :ledger type :type key :key} params
+                        ledger        (keyword network ledger)
```

Comment:
Should this be `(keyword network dbid)` ?

Reply:
Good catch, thanks.